website/docs/community/templates.mdx

```diff
@@ -48,6 +48,7 @@ If you are unsure about a template, inspect `package.json` and `wails.json` for
 ## Elm
 
 - [wails-elm-template](https://github.com/benjamin-thomas/wails-elm-template) - Develop your GUI app with functional programming and a **snappy** hot-reload setup :tada: :rocket:
+- [wails-elm-tailwind-template](https://github.com/rnice01/wails-elm-tailwind-template) - Combine the powers :muscle: of Elm + Tailwind CSS + Wails! Hot reloading supported.
```

Comment:
It is recommended to use the naming format of `wails-template-xxx`

Reply:
Did you want me to just update the link text or rename the whole repo?

Reply:
Maybe both, thanks.

Reply:
@rnice01 - Any updates? Would *love* to get this in 👍

Reply:
@misitebao @leaanthony 
It's been updated, thanks for your patience!